migration-blob-storage.tf

```
@@ -28,14 +28,11 @@ moved {
 resource "azurerm_storage_blob" "migration-st" {
   count                  = local.is_migration_environment ? 1 : 0
   name                   = "${var.product}-migration-blob-st-${var.env}"
-  storage_account_name   = module.sa-migration[0].storageaccount_name
+  storage_account_name   = "sa${var.env}${var.product}mig02"
```

```suggestion
  storage_account_name   = "module.sa-migration-standard[0].storageaccount_name"
```

To avoid the issue where TF will try to create the blob before the storage account exists if you run this in a "clean" environment or need to re-deploy the storage account